Dockerfile

```diff
@@ -1,15 +1,26 @@
+FROM node:16 AS builder
```

Comment:
We can probably use the `-slim` image, let's do that while we're at it

Reply:
Done.